backend/lib/spree_backend.rb

```diff
@@ -1,3 +1 @@
 require 'spree/backend'
-require 'sprockets/rails'
-require 'bootstrap'
```

Comment:
I noticed `bootstrap` and `sprockets/rails` removed from the required list on the backend, could this be done on the front end also?

```ruby
require 'spree_core'
require 'spree_api'

require 'glyphicons'
require 'inline_svg'
require 'canonical-rails'
require 'jquery-rails'
require 'turbolinks'
require 'responders'

require 'spree/frontend/middleware/seo_assist'
require 'spree/frontend/engine'
```

Reply:
NOTE:  poppers js is also removed from this list, all seems to work fine on my localhost:3000.

Reply:
@MatthewKennedy that bootstrap requirement statement was duplicated - https://github.com/spree/spree/pull/11041/files#diff-cb942b864401de208e3140b508d1d50741dbb927a7e28b1b31bf240beb9cf073R10

Reply:
Same as the popper_js